docs/user-manual/en/using-server.md

```diff
@@ -53,6 +53,8 @@ The following highlights some important folders on the distribution:
 
 ## Creating a Broker Instance
 
+Before the broker is used, an broker *instance* must be created.
```

Comment:
an -> a

I would put the sentence at the start of the existing paragraph below about broker instances.